pool/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/killa-beez/gopkgs/pool
+
+go 1.13
```

Comment:
Can you change this to 1.12?  This limits older versions from using the package.

`go mod init` sets it to whatever version you are currently using.  I've been manually changing them. I don't know if there is a better way to handle it.